client/mm/libxc/binance.go

```diff
@@ -553,30 +556,42 @@ func (bnc *binance) setBalances(ctx context.Context) error {
 	return nil
 }
 
-// setTokenIDs stores the token IDs for which deposits and withdrawals are
-// enabled on binance.
-func (bnc *binance) setTokenIDs(coins []*bntypes.CoinInfo) {
+// readCoins stores the token IDs for which deposits and withdrawals are
+// enabled on binance and sets the minTransfers map.
+func (bnc *binance) readCoins(coins []*bntypes.CoinInfo) {
 	tokenIDs := make(map[string][]uint32)
+	minWithdraw := make(map[uint32]map[uint32]uint64)
```

Comment:
This could just be a `map[uint32]uint64` since the dex asset ID implies the network.